pkg/processor/runtime/nodejs/js/wrapper.js

```diff
@@ -0,0 +1,265 @@
+/*
+Copyright 2017 The Nuclio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const net = require('net')
+const events = require('events')
+
+const jsonCtype = 'application/json'
+const initContextFunctionName = 'initContext'
+
+let context = {
+    userData: {},
+    callback: async (handlerResponse) => {
+        context._eventEmitter.emit('callback', handlerResponse)
+    },
+    Response: Response,
+    logger: {
+        error: (message, withData) => log('error', message, withData),
+        warn: (message, withData) => log('warning', message, withData),
+        info: (message, withData) => log('info', message, withData),
+        debug: (message, withData) => log('debug', message, withData),
+        errorWith: (message, withData) => log('error', message, withData),
+        warnWith: (message, withData) => log('warning', message, withData),
+        infoWith: (message, withData) => log('info', message, withData),
+        debugWith: (message, withData) => log('debug', message, withData),
+    },
+    _socket: undefined,
+    _eventEmitter: new events.EventEmitter(),
+}
+
+function Response(body = null,
+                  headers = null,
+                  contentType = 'text/plain',
+                  statusCode = 200) {
+    this.body = body
+    this.headers = headers
+    this.content_type = contentType
+    this.status_code = statusCode
+
+    if (!isString(this.body)) {
+        this.body = JSON.stringify(this.body)
+        this.content_type = jsonCtype
+    }
+}
+
+function log(level, message, withData) {
+    if (withData === undefined) {
+        withData = {}
+    }
+
+    const record = {
+        datetime: new Date().toISOString(),
+        level: level,
+        message: message,
+        with: withData,
+    }
+    context._socket.write('l' + JSON.stringify(record) + '\n')
+}
+
+function isString(obj) {
+    return typeof (obj) === 'string' || (obj instanceof String)
```

Comment:
I understand, but since it is being used to resolve the user's function output (you might have a nuclio function that return that very `new String('blabla')`, then Im left with no choice and to support that as well.